ecs_fargate/README.md

```diff
@@ -7,30 +7,30 @@ Get metrics from all your containers running in ECS Fargate:
 * CPU/Memory usage & limit metrics
 * Monitor your applications running on Fargate via Datadog integrations or custom metrics.
 
-## Setup
+The Datadog agent retrieves metrics for the task definition's containers via the ECS Task Metadata endpoint, `169.254.170.2/v2/stats`. According to the [ECS Documentation][18] on that endpoint:
```

Comment:
Can we link to the ECS Task Metadata  docs but not the IP?

Reply:
Also we should focus more on what the integration enables you to do over . pure cloud watch, rather than the how.  Eg:  -autodiscovery, collect container metrics,  resource metrics, tracing.

Reply:
👍